book/website/index.md

```diff
@@ -1,75 +1,124 @@
 (welcome)=
 # Welcome
 
-*Welcome to The Turing Way handbook to reproducible, ethical and collaborative data science.*
+_Welcome to The Turing Way Handbook to Reproducible, Ethical, and Collaborative Data Science._
 
-_The Turing Way_ is an open science, open collaboration, and community-driven project.
-We involve and support a diverse community of contributors to make data science accessible, comprehensible and effective for everyone.
-Our goal is to provide all the information that researchers, data scientists, software engineers, policymakers, and other practitioners in academia, industry, government and the public sector need to ensure that the projects they work on are easy to reproduce and reuse.
+_The Turing Way_ is an open source book, a global collaboration and a supportive community.
```

Comment:
I'm kind of sad to lose "community-driven" in this edit - what's the rationale behind that change @malvikasharan?

Reply:
Me too. I am sorry that I have not been following this thread until now. Let me do the review.

Reply:
```suggestion
_The Turing Way_ is an open science, open collaboration, and community-driven project. 
```